profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -99,6 +100,16 @@ public static RfElementConfiguration createWithDefaults(RadioType radioType) {
     	return ret;
     }
     
+    public static RfElementConfiguration createWithDefaults(RadioType radioType, ApModel apModel) {
+		RfElementConfiguration ret = createWithDefaults(radioType);
```

Comment:
RfElementConfiguration object is radio-specific. If you add radioType as a private member attribute that's set here then there won't be a need for any public member methods which have a radioType parameter. This is the same approach used in ElementRadioConfig. Specifically, it means the radioType parameter defined for getMinAutoCellSize can be removed. 

Similar static maps and getters may also be needed to replace the public DEFAULT_ constants

Reply:
Are you able to add radioType as a member attribute?

Reply:
Yes I'll add that. 
I've also adjust the logic in getMinAutoCellSize to reflect that as well.